.pytool/Plugin/DependencyCheck/DependencyCheck.py

```diff
@@ -94,7 +94,7 @@ def RunBuildPlugin(self, packagename, Edk2pathObj, pkgconfig, environment, PLM,
         for file in INFFiles:
             ip = InfParser()
             logging.debug("Parsing " + file)
-            ip.SetEdk2Path(Edk2pathObj).ParseFile(file)     # MU_CHANGE
+            ip.SetBaseAbsPath(Edk2pathObj.WorkspacePath).SetPackagePaths(Edk2pathObj.PackagePathList).ParseFile(file)

```

Comment:
This would seem to already be in 2405, looking at the original commit. It also does not do what the PR message says. I suspect this can be dropped, @makubacki?

Reply:
Looking at the history you're correct that this change isn't necessary.  I'll fix it now.